### ISSUE TITLE
Missing edge gateway commands in vcd-cli

### ISSUE THREAD

**Issue by pasikarkkainen** [vmware/vcd-cli#100]
It seems vcd-cli is missing commands to manage Edge Gateways. Are those on the roadmap? 

older vca-cli can manage edge gateways.

**Comment by pacogomez** [comment]
Yes, we have support for edge gateway in the roadmap of the new `vcd-cli`.

**Comment by pasikarkkainen** [comment]
Great, good to hear! Is there any estimated timeframe for these?

**Comment by gmile** [comment]
Looking forward to seeing this too. Right now we have to go and manually add NAT rules :(

**Comment by akononykhin** [comment]
Hello

Could you say when the commands for managing Edge Gateways will be back added?
We really need them! Honestly these are the main functionality we need vcd-cli for :(

**Comment by hodgesrm** [comment]
Commands are in the roadmap for the next vCD release cycle (after the current one which is currently named version 9.5). We'll be implementing them during the second half of the year. 

We share the urgency about getting full network feature support in but have prioritized getting documentation and test in good order first so we can add new features more quickly thereafter. 

We would welcome anyone who wants to dig in and add underlying support in pyvcloud before we get to it. That's the hard part--adding vcd-cli commands is mostly just surface logic to process options.

**Comment by pascalsaul** [comment]
Still missing all commands to manage everything under the "Load Balancer" tab. Any plans?

**Comment by pandeys** [comment]
It is not included in current release. You can contribute and write functionality.
